packages/main/src/Input.js

```diff
@@ -693,8 +693,19 @@ class Input extends UI5Element {
 		}
 	}
 
+	/**
+	 * Fire only 'change' event if it is triggered by focusout.
+	 * Otherwise, fire a "submit" event also.
+	 *
+	 * @private
+	 */
 	_handleChange(event) {
 		this.fireEvent(this.EVENT_CHANGE);
+
+		if (this._isSubmit) {
+			this.fireEvent(this.EVENT_SUBMIT);
+			this._isSubmit = false;
```

Comment:
Could you describe and initialise this._isSubmit in the constructor, you can see that there are other private fields there and as the variable is used in more than one method we have to rename it to something more descriptive, such as this.submitAction. Otherwise, looks good.